src/containers/pages/IRSLite/MapLite/helpers.ts

```diff
@@ -0,0 +1,269 @@
+import { STATUS_HEADER } from '../../../../configs/lang';
+import { circleLayerConfig, fillLayerConfig, lineLayerConfig } from '../../../../configs/settings';
+import { FEATURE } from '../../../../constants';
+import { STRUCTURE_LAYER } from '../../../../constants';
+import { GenericJurisdiction } from '../../../../store/ducks/generic/jurisdictions';
+import { StructureFeatureCollection } from '../../../../store/ducks/generic/structures';
+
+/** The default indicator stop */
+export const defaultIndicatorStop = [
+  ['Sprayed', TASK_GREEN],
+  ['Not Sprayed', TASK_RED],
+  ['Partially Sprayed', TASK_ORANGE],
+  ['Not Visited', TASK_YELLOW],
+  ['Not Eligible', BLACK],
+];
+
+/** IRS Indicator stops
+ * These are all the indicator stops for IRS that we know about.
+ */
+export const IRSIndicatorStops: { [key: string]: string[][] } = {
+  namibia2019: defaultIndicatorStop,
+  zambia2019: [
+    ['Complete', TASK_GREEN],
+    ['Not Sprayed', TASK_RED],
+    ['Partially Sprayed', TASK_GREEN],
+    ['Not Visited', TASK_YELLOW],
+    ['Not Eligible', BLACK],
+  ],
+};
+
+/** IRS Lite Indicator stops
+ * These are all the indicator stops for IRS Lite that we know about.
+ */
+export const IRSLiteIndicatorStops: { [key: string]: string[][] } = {
+  zambia2020: defaultIndicatorStop,
+};
+
+/** interface to describe and indicator row item */
+export interface IndicatorRowItem {
+  denominator: string | number;
+  description: string;
+  listDisplay?: Dictionary | string;
+  numerator: string | number;
+  title: string;
+  unit?: string;
+  value: string | number;
+}
+
+/** the indicator row type */
+export type IndicatorRows = IndicatorRowItem[];
+
+/** IRS Indicator rows
+ * These are all the indicator rows for IRS that we know about.
+ */
+export const IRSIndicatorRows: { [key: string]: IndicatorRows } = {
+  zambia2020: [
+    {
+      denominator: 'totstruct',
+      description: 'Percent of Total Structures Sprayed',
+      numerator: 'sprayed',
+      title: 'Spray Coverage (Effectiveness)',
+      value: 'spraycov',
+    },
+    {
+      denominator: 'targstruct',
+      description: 'Percent of Targeted Structures Sprayed',
+      numerator: 'sprayed',
+      title: 'Targeted Spray Coverage (Effectiveness)',
+      value: 'spraycovtarg',
+    },
+    {
+      denominator: 'targstruct',
+      description: 'Percent of found structures over targeted',
+      numerator: 'found',
+      title: 'Found Coverage',
+      value: 'spraycovtarg',
+    },
+    {
+      denominator: 'found',
+      description: 'Percent of structures sprayed over found',
+      numerator: 'sprayed',
+      title: 'Spray Success Rate (PMI SC)',
+      value: 'spraysuccess',
+    },
+  ],
+};
+
+/** Utility function to build structure layer
+ * @param {StructureFeatureCollection} structures - Feature Collection of structures
+ * @param {string[][]} indicatorStops - the indicator stops
+ */
+export const structuresLayerBuilder = (
```

Comment:
This function should be removed. It is no longer used after doing away with `GisidaWrapper`.

Reply:
Done